MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: Data annotations in Maui DataForm control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in Maui DataForm (SfDataForm)
+
+The data form supports the following attribute, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name] `Link`
+</td>
+<td>
+Specifies the label text.
+</td>
+</tr>
+<tr>
+<td>
+{{'[GroupName]`Link`
+</td>
+<td>
+Specifies the group name which groups the fields in the data form. Refer to  `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShortName] `Link`
+</td>
+<td>
+Specifies the label text. It takes higher priority than Name.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateField]`Link`
+</td>
+<td>
+Specifies whether the field should be auto generated or not. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateFilter]`Link`
+</td>
+<td>
+Specifies the value that indicates whether filtering UI is automatically displayed for that field.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ResourceType]`Link`
+</td>
+<td>
+Specifies the type that contains the resources for the other disply attributes property.
```

Comment:
Modify content, that it is used to localize the label content

Reply:
Content modified